full/src/test/java/apoc/load/LoadCsvTest.java

```diff
@@ -203,6 +223,110 @@ public void testLoadCsvByUrl() throws Exception {
 
     }
 
+    @Test
+    public void testLoadCsvWithUserPassInUrl() {
+        String userPass = "user:password";
+        String token = Util.encodeUserColonPassToBase64(userPass);
+
+        new MockServerClient("localhost", 1080)
+                .when(
+                        request()
+                                .withPath("/docs/csv")
+                                .withHeader("Authorization", "Basic " + token)
+                                .withHeader("\"Content-type\", \"text/csv\""),
+                        exactly(1))
+                .respond(
+                        response()
+                                .withStatusCode(200)
+                                .withHeaders(
+                                        new Header("Content-Type", "text/csv; charset=utf-8"),
+                                        new Header("Cache-Control", "private, max-age=1000"))
+                                .withBody("headFoo,headBar\n" + "one,two\n" + "three,four\n" + "five,six\n")
```

Comment:
Anche qui fai una lista di mappe e poi trasformala in stringa e usala per controllare i valori sotto